vhdbuilder/packer/install-dependencies.sh

```diff
@@ -67,6 +123,13 @@ RuntimeMaxUse=1G
 ForwardToSyslog=yes
 EOF
 
+record_benchmark 'Install Dependencies (Lines 102-124) End'
+stop_watch 'Install Dependencies'
+#Benchmark 3 End
```

Comment:
These comments like start and end can also be removed in the PR before merging -- it's ok for testing though